data/human/navy missions intro.txt

```diff
@@ -0,0 +1,1496 @@
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+# Republic/Navy Campaign Project Lead: Bereskatuket
+# Head Writer/Creative Consultant/Resident Grammar Nazi: Warlord Mike
+# Opening Missions Writing Credit to: Disiuze, Kiko, Moonbeam, Kat
+# 
+
+event "enlistment announced"
+	date 11 7 3014
+
+
+mission "enlistment announced"
+	landing
+	source
+		government "Republic"
+	to offer
+		has "event: enlistment announced"
+	on offer
+		log "The Republic Navy announced it will begin recruiting."
+		conversation
+			`On behalf of the republic the navy will begin accepting volunteers into navy service next week. With the loss of ships in the devastating nuclear attack on our ship yard we need all combat capable ship captains. Volunteers will be paid a per diem. Interested ship captains will be required to prove their worth to earn their place. More information will be provided at a later point in time.`
+
+
+event "enlistment begins"
+	date 18 7 3014
+
+mission "Republic Bounty 1"
+	name "Republic Bounty Hunting"
+	description `Hunt down the <npc>, a pirate ship that has been assaulting miners in the Boral system.`
+	minor
+	source
+		government "Republic"
+		attributes "dirt belt"
+	to offer
+		has "event: war begins"
+		"combat rating" > 40
+		not "chosen sides"
+		random < 65
+
+	on offer
+		conversation
+			`You barely have time to find your bearings in this spaceport when you're approached by a man in a crisp, navy blue Republic uniform.`
+			`	"You have a face that says you're looking for work," he says, in a surprisingly cheerful tone. "We're in need of someone to deal some pirates that have been causing trouble near Boral. Not normally something we'd contract out, but, luckily for you, we're a little short on free ships right now."`
+			choice
+				`	"Sure."`
+				`	"Sorry, I'm not really into bounty hunting."`
+					defer
+			`	"Much obliged," he says. "It's a ship called the <npc>. It's a small little devil, but don't let it take you off guard."`
+				accept
+	on accept
+		log `Hunting a pirate ship called the "<npc>," which has been plundering ships near Boral.`
+
+	npc kill
+		personality heroic staying nemesis target
+		government Bounty
+		system
+			near Boral 1 2
+		ship "Clipper (Heavy)" "Black Fang"
+		dialog `The <npc> has been eliminated. You can claim the bounty payment by returning to <destination>.`
+
+	on complete
+		payment 175000
+		dialog `Upon landing, the commander greets you, seemingly surprised by your return. "The Republic needs more captains like you. Hopefully this payment of <payment> will be enough to convince you to help us out the next time." You nod. "Captain <last>, correct?" he says, and barely gives you enough time to agree before he continues speaking. "Right. I'm Commander Kent, in case I didn't introduce myself. I'll be keeping an eye out for you in case we ever need more assistance." `
```

Comment:
It can reflect that, in a broad way, the Navy is getting a little desperate. Hiring hiring merchants with little history certainly backs that up, and while the Navy usually has a lot of decorum, for some personnel, that could slip a bit. Although it probably should either be a little more exaggerated, or cut down to be more professional, yeah.